packages/plugins/src/foundry/get-bundle-info.ts

```diff
@@ -102,16 +104,18 @@ const broadcasting = args[2] === 'true'
       writeCanonicalConfig(canonicalConfigFolder, configUri, canonicalConfig)
 
       const ipfsHash = configUri.replace('ipfs://', '')
-      const cachePath = path.resolve('./cache')
+      const artifactCachePath = path.resolve(
+        '.canonical-configs/configArtifacts'
```

Comment:
We'd be putting them in the foundry cache actually since we pull that from the config file. Yeah, it's probably better to put them there since the user shouldn't be committing these to version control anyway.